src/cmsHttp.ts

```diff
@@ -116,6 +116,10 @@ class CmsClient {
 
     await this.http({ zoneId, url: content.tracker });
   }
+
+  async cmsAction(request: AxiosRequestConfig) {
+    return this.axios.request(request);
```

Comment:
nit: not sure this function is totally necessary

Reply:
alternatively, I think just `request` is a better name for this

Reply:
yeah, it had more meat in it when I started and eventually became this. will just remove it.